app/src/test/java/com/example/tanya/godeltechchallengeandroid/domain/interactor/SetStartStatusTest.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.tanya.godeltechchallengeandroid.domain.interactor
+
+import com.example.tanya.godeltechchallengeandroid.data.prefs.ApplicationRepository
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.MockitoAnnotations
+
+class SetStartStatusTest {
+
+    @Mock
+    private lateinit var applicationRepository: ApplicationRepository
+    private lateinit var setStartStatus: SetStartStatus
+
+    private val testValue = true
+
+    @Before
+    fun setUp() {
+        MockitoAnnotations.initMocks(this)
+        setStartStatus = SetStartStatus(applicationRepository)
+    }
+
+    @Test
+    fun onSetStartStatus_shouldUpdateApplicationRepository() {
+        setStartStatus.execute(testValue)
```

Comment:
Why don't you test the return value of the SetStartStatus instance? You have to check it completes successfully.